google_guest_agent/agentcrypto/mtls_mds.go

```diff
@@ -0,0 +1,59 @@
+// Package agentcrypto provides various cryptography related utility functions for Guest Agent.
+package agentcrypto
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/GoogleCloudPlatform/guest-agent/google_guest_agent/uefi"
+	"github.com/GoogleCloudPlatform/guest-logging-go/logger"
+)
+
+const (
+	googleGUID                 = "8be4df61-93ca-11d2-aa0d-00e098032b8c"
```

Comment:
Can you please document each of these constants? Use the perspective of a system user (If I were to perform this steps manually what would mean each of these "artifacts"?).

Reply:
Done